components/automate-ui/src/app/services/user-preferences/user-preferences.actions.ts

```diff
@@ -0,0 +1,64 @@
+import { HttpErrorResponse } from '@angular/common/http';
+import { Action } from '@ngrx/store';
+import {
+  UserPreferencesPayload, UserPreferenceResponse
```

Comment:
align these two imports  in separate line like :
https://github.com/chef/automate/blob/b8486c44deeb30362f8691fe1fa5f5025d60e042/components/automate-ui/src/app/modules/infra-proxy/infra-roles/infra-roles.component.ts#L1-L3

Reply:
done